### PR TITLE
Nasc/per role compute and storage pg

### PR DESCRIPTION
**Pull request by nasc17**
Adding in per role configuration edits to Compute and Storage page. This section will remain hidden until backend is able to be called. 

coordinatorconifgcs

### CODE REVIEW

File: extensions/arc/src/localizedConstants.ts [hunk 1]
```diff
@@ -150,9 +151,11 @@ export const computeAndStorageDescriptionPartSix = localize('arc.computeAndStora
 export const node = localize('arc.node', "node");
 export const nodes = localize('arc.nodes', "nodes");
 export const workerNodes = localize('arc.workerNodes', "Worker Nodes");
+export const coordinatorNode = localize('arc.coordinatorNode', "Coordinator Node");
 export const storagePerNode = localize('arc.storagePerNode', "storage per node");
 export const workerNodeCount = localize('arc.workerNodeCount', "Worker node count:");
 export const configurationPerNode = localize('arc.configurationPerNode', "Configuration (per node)");
+export const configurationCoordinatorNode = localize('arc.configurationCoordinatorNode', "Configuration");
```
Comment (Charles-Gagnon) [inline extensions/arc/src/localizedConstants.ts:158]: nit: just name this `arc.configuration`. You can reuse strings and it makes it easier to find and re-use if you keep it generic

File: extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts [hunk 2]
```diff
@@ -448,32 +564,88 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 		return configurationSection;
 	}
 
-	private editCores() {
+	private editWorkerCores() {
```
Comment (Charles-Gagnon) [inline extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts:567]: return type for these and the other functions below

**Review by Charles-Gagnon — APPROVED** [review]

File: extensions/arc/src/localizedConstants.ts (hunk 1 above)
Comment (nasc17) [reply extensions/arc/src/localizedConstants.ts:158]: meant to do that haha

**Comment by bergeron** [comment]
@nasc17 While we're keeping the per-role stuff hidden, does the page look and act like it did before?  Any changes?

**Comment by nasc17** [comment]
> @nasc17 While we're keeping the per-role stuff hidden, does the page look and act like it did before? Any changes?

Correct, I did not add the coordinator section to the content container so that is not seen or accessed at all. The azdata calls are the same as before. Comments were added just to bring attention to needing to change them  in the future

**Review by bergeron — APPROVED** [review]

**Pull request merged**